kiigame.js

```diff
@@ -120,91 +27,174 @@ export class KiiGame {
             );
         }
 
+        // Alternative variable for `this` to allow reference even when it's shadowed
+        var self = this;
+
+        // The amount of rewards found. LZ specific, TODO refactor
+        this.rewards = 0;
+
+        // Define variables from stage for easier use
```

Comment:
These comments seem to be left here by accident. The stage-related variables are not defined at this part.

```suggestion
```